jbpm-workitems/jbpm-workitems-rest/src/test/java/org/jbpm/process/workitem/rest/RestWorkItemHandlerTest.java

```diff
@@ -1069,7 +1069,29 @@ public void testHeadersSingleValue() {
         RESTWorkItemHandler handler = new RESTWorkItemHandler();
         String headerKey = "headerKey";
         String headerValue = "headerValue";
-        String headers = headerKey + "=" + headerValue;
+        String headers = headerKey + ":" + headerValue;
+         
+        WorkItemImpl workItem = new WorkItemImpl();
+        workItem.setParameter("Url",
+                              serverURL + "/header/" + headerKey);
+        workItem.setParameter("Method",
+                              "GET");
+        workItem.setParameter(PARAM_HEADERS, headers);
+        
+        WorkItemManager manager = new TestWorkItemManager();
+        handler.executeWorkItem(workItem,
+                                manager);
+        Map<String, Object> results = ((TestWorkItemManager) manager).getResults(workItem.getId());
+        String result = (String) results.get(PARAM_RESULT);
+        assertEquals(result, headerValue);
```

Comment:
It should be the other way around (expected, current):
```suggestion
        assertEquals(headerValue, result);
```

Reply:
@gmunozfe Made suggested changes in this test case and in other test cases as well.